src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java

```diff
@@ -148,6 +149,14 @@
     private transient RunningJobs runningJobs = new RunningJobs();
     private List<GerritProject> gerritProjects;
     private List<GerritProject> dynamicGerritProjects;
+    //! This latch will be used to signal that the project list is ready for use.
+    //! For static configuration, this will be ready immediately.
+    //! For dynamic configuration, this will be ready after the first time that
+    //! the project list has been fetched.
+    //!
+    //! Default the latch to the non-waiting zero state, which corresponds to
+    //! static project configurations.
+    private CountDownLatch projectListIsReady = new CountDownLatch(0);
```

Comment:
This might need to be `volatile` and `transient` and then created anew in `readResolve()`.

Reply:
I went with `transient` and just made sure to initialize it in the constructors.

Reply:
Looks like there's still a situation where this can be `null` _and_ the EventListener thread is running.  Looking into where things are getting constructed to see why this is.

Reply:
Okay, doing that in `readResolve` seems to have solved the problem.  It looks like the first run of the plugin is from `readResolve`, and neither of the constructors is called, since the deserializer will only call a zero-argument version of the constructor.